src/charts/legend.js

```diff
@@ -333,17 +372,21 @@ define(function(require){
                 .style('font-size', `${textSize}px`)
                 .style('letter-spacing', `${textLetterSpacing}px`);
 
-            svg.select('.legend-group')
-                .selectAll('g.legend-line')
-                .selectAll('g.legend-entry')
-              .append('text')
-                .classed('legend-entry-value', true)
-                .text(getFormattedQuantity)
-                .attr('x', chartWidth - valueReservedSpace)
-                .style('font-size', `${textSize}px`)
-                .style('letter-spacing', `${numberLetterSpacing}px`)
-                .style('text-anchor', 'end')
-                .style('startOffset', '100%');
+            if (hasQuantities) {
+                svg.select('.legend-group')
```

Comment:
I think it would make sense to transform the section of code in the `hasQuantities` statement to a function with a name that describes what this section of code is doing. That way it makes it consistent since the other `else` case does that.

Reply:
Cleaner, yes. Done